cast5/tests/lib.rs

```diff
@@ -0,0 +1,77 @@
+#![no_std]
+extern crate block_cipher_trait;
+#[macro_use]
+extern crate generic_array;
+extern crate cast5;
+
+use block_cipher_trait::BlockCipher;
+use cast5::Cast5;
+use generic_array::GenericArray;
+
+#[test]
+fn single_plaintext_key_ciphertext_sets() {
```

Comment:
It could be nice to also add additional test vectors, e.g. from [here](https://www.cosic.esat.kuleuven.be/nessie/testvectors/bc/cast-128/Cast-128-128-64.verified.test-vectors) and pack them into `blb` format used by test macro from `block-cipher-trait`. But I guess it will be faster for me to do it after we'll merge this PR.

Reply:
Good idea, is there some docs on how to do the packing?

Reply:
Documentation is a bit scarce right now. `blobby` docs [describe](https://docs.rs/blobby/) format itself, but converting from (and to) text files is done via small [tool](https://github.com/RustCrypto/utils/blob/master/blobby/examples/convert.rs) in `examples/` directory.

Reply:
done